README.md

```diff
@@ -1,3 +1,14 @@
+# **THIS IS A FORK**
+
+The upstream library needs to be built and **installed on the build machine**. This verion of the library does not need to be installed.
```

Comment:
This is not true! You can include the repository as submodule or clone it and simply include its path to your CMake project to use it without installing.

Reply:
Not sure what I was running into yesterday... The build via FetchContent and intellisense are both working for me now. Thanks for dropping in and making me questions my assumptions @lfreist 👍